programs.cfg

```diff
@@ -0,0 +1,41 @@
+{
+    ["gerti-client"] = {
+        files = {
+            ["master/GERTi/GERTiClient.lua"] = "/lib",
+        },
+        name = "GERTi Client",
+        description = "Allows a computer to access the GERTi network.",
+        authors = "MajGenRelativity",
+        repo = "tree/master/GERTi",
+    },
+    ["gerti-mnc"] = {
+        files = {
+            ["master/GERTi/GERTiMNC.lua"] = "/bin",
+        },
+        name = "GERTi Master Network Controller",
+        description = "Master Network Controller for GERTi networks. Install gerte-api for internetwork support.",
+        authors = "MajGenRelativity",
+        repo = "tree/master/GERTi",
+    },
+    ["gerte-api"] = {
+        files = {
+            ["master/GERTe/GERTeAPI.lua"] = "//lib",
+        },
+        name = "GERTe Internetwork API",
+        description = "Allows access to other GERTi networks over the internet.",
```

Comment:
It allows access to anything running a GERTe client. This is not limited to other GERTi networks